stdlib/public/core/Span/RawSpan.swift

```diff
@@ -45,15 +49,26 @@ extension RawSpan {
   ///   - buffer: an `UnsafeRawBufferPointer` to initialized memory.
   ///   - owner: a binding whose lifetime must exceed that of
   ///            the newly created `RawSpan`.
+  @_disallowFeatureSuppression(NonescapableTypes)
   @_alwaysEmitIntoClient
+  @lifetime(immortal)
   public init(
     _unsafeBytes buffer: UnsafeRawBufferPointer
-  ) -> dependsOn(immortal) Self {
+  ) {
     self.init(
       _unchecked: buffer.baseAddress, byteCount: buffer.count
     )
   }
 
+  @_disallowFeatureSuppression(NonescapableTypes)
+  @_alwaysEmitIntoClient
+  @lifetime(immortal)
```

Comment:
Should be `@lifetime(buffer)`. The resulting Span depends on the validity of the buffer.